From 2945643021938af3d9b06b5e527d46e4dd8887ae Mon Sep 17 00:00:00 2001
From: "Valery V. Vorotyntsev" <valery_vorotyntsev@xyratex.com>
Date: Fri, 17 Aug 2012 00:06:06 +0300
Subject: [PATCH 288/370] Makefile.am (SUBDIRS_LIBS): restore the empty line
 after `fid'

The order of entries of SUBDIRS_LIBS does matter. Move the comment
closer to the variable being described. Restore the empty line,
which is used as a separator.
---
 Makefile.am | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Makefile.am b/Makefile.am
index 3b5f99d..152412b 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -1,9 +1,8 @@
 SUBDIRS_BASE = m4 doc man patches
 
-# Order is important until fid is reached. Fop needs all libs
+# Order is important until "fid" is reached. "fop" needs all libs
 # before it in the built list.
 # Keep this list alphabetically sorted after "net".
-
 SUBDIRS_LIBS =                     \
                lib                 \
                xcode/ff2c          \
@@ -11,6 +10,7 @@ SUBDIRS_LIBS =                     \
                fol                 \
                db                  \
                fid                 \
+                                   \
                addb/addbff         \
                net                 \
                fop                 \
-- 
1.8.3.2

